pkg/datapath/linux/config/config.go

```diff
@@ -204,6 +206,25 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 	cDefinesMap["TRACE_PAYLOAD_LEN"] = fmt.Sprintf("%dULL", option.Config.TracePayloadlen)
 	cDefinesMap["MTU"] = fmt.Sprintf("%d", cfg.MtuConfig.GetDeviceMTU())
 
+	ciliumHost, err := netlink.LinkByName("cilium_host")
```

Comment:
Note, there's an ordering consideration here:

`Daemon.init()` calls:
-> `d.createNodeConfigHeaderfile(...)`
    -> `d.datapath.WriteNodeConfig(...)`
-> ...
-> `d.Datapath().Loader().Reinitialize(...)`
    ->  `setupBaseDevice(...)`

In summary, the device may not exist until later.

Reply:
I think I solved that with the last push and 1d0f6a80b9bd734a28cc3439eb923f14ebe6dd94. Thanks for pointing that out!